bin/parse_podfile_lock

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require 'yaml'
+
+podfile_lock = begin
+  YAML.safe_load(File.read(ARGV[0]))
+               rescue SyntaxError, Psych::SyntaxError
```

Comment:
`YAML.safe_load` throws more types of errors than the two above, like `Psych::DisallowedClass`, `Psych::BadAlias`, etc... 
It would be better to just `rescue StandardErrors`